bin/compile

```diff
@@ -147,6 +147,10 @@ export LIBRARY_PATH=/app/.heroku/vendor/lib:$BUILD_DIR/.heroku/vendor/lib:/app/.
 export LD_LIBRARY_PATH=/app/.heroku/vendor/lib:$BUILD_DIR/.heroku/vendor/lib:/app/.heroku/python/lib:$LD_LIBRARY_PATH
 export PKG_CONFIG_PATH=/app/.heroku/vendor/lib/pkg-config:$BUILD_DIR/.heroku/vendor/lib/pkg-config:/app/.heroku/python/lib/pkg-config:$PKG_CONFIG_PATH
 
+if [[ $VCAP_SERVICES == *appdynamics* ]]; then
+  source $BIN_DIR/steps/appdynamics-setup
+fi
+
```

Comment:
It would probably be better if the "if" deciding whether to install AppDynamics module is in the `steps/appdynamics-setup` script, not the `compile` script

We'd also recommend that this script be sourced with the others (starting L220 through L247)